public/v2/apps/collabora-online.json

```diff
@@ -0,0 +1,57 @@
+{
+  "captainVersion": "2",
+  "documentation": "Inspired by https://github.com/caprover/one-click-apps/pull/111 and from https://www.collaboraoffice.com/code/docker/",
+  "displayName": "Collabora Online",
+  "description": "Collabora Online is an online and collaborating office suite",
+  "dockerCompose": {
+    "version": "1",
+    "services": {
+      "$$cap_appname": {
+        "image": "collabora/code:$$cap_collabora_version",
+        "containerHttpPort": "9980",
+        "restart": "always",
+        "environment": {
+          "domain": "$$cap_wopihost_domain",
+          "cert_domain": "$$cap_appname.$$cap_root_domain",
+          "server_name": "$$cap_appname.$$cap_root_domain",
+          "extra_params": "--o:ssl.enable=false --o:ssl.termination=true",
+          "username": "$$cap_admin_user",
+          "password": "$$cap_admin_pass"
+        }
+      }
+    }
+  },
+  "instructions": {
+    "start": "Collabora CODE version is an online Office Suite. \nWhen you are using Nextcloud or ownCloud you can use it to enhance your online office document experience.\nThe install will take about a minute for the process to finish.",
+    "end": "Collabora is deployed and available as $$cap_appname.\n\nIMPORTANT: You need do 5 manual steps before access.\n\n 1 - enable websocket on HTTP settings tab\n\n 2 - add the \"proxy_read_timeout 36000s;\" instruction below \"proxy_http_version 1.1;\"\n\n3 - Enable and force HTTPS and wait few minutes.\n\n4 - add the Collabora Online extension for Nextcloud (3.7.0 for now)\n\n5 - go to your Nextcloud plugin configuration > collabora online > use your own server > paste the FQDN of your new instance of collabora > save\n\nThat's all, you can use collabora online to all your Nextcloud instance."
+  },
+  "variables": [{
+    "id": "$$cap_admin_user",
+    "label": "Admin user",
+    "description": "permit access to the web admin interface",
+    "defaultValue": "adminuser",
+    "validRegex": "/^([a-zA-Z0-9])+$/"
+  },
+    {
+      "id": "$$cap_admin_pass",
+      "label": "Admin password",
+      "description": "admin password",
+      "validRegex": "/.{1,}/"
+    },
+    {
+      "id": "$$cap_collabora_version",
+      "label": "Collabora CODE Version",
+      "defaultValue": "4.2.4.4",
+      "description": "Checkout their docker page for the valid tags https://hub.docker.com/r/collabora/code/tags/",
+      "validRegex": "/^([^\\s^\\/])+$/"
+    },
+    {
+      "id": "$$cap_wopihost_domain",
+      "label": "WOPI host domain",
+      "defaultValue": "example\\.domain\\.tld",
+      "TODO" : "rename default value with better example",
```

Comment:
Change the default to `$$cap_appname.$$cap_root_domain` ?

Reply:
I changed the description and thz default value (this is the nextcloud url and not the url of the code instance)